app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,220 @@
+   /*border: 1px solid red;*/
+   border-collapse: collapse;
+   margin: 0rem auto;
+ }
+
+ table tr {
+   border-bottom: 1px double rgb(106, 108, 111);
+ }
+
+ tbody {
+   overflow: auto;
+   width: 100%;
+ }
+
+ table tr:nth-child(even) {
+  background-color: rgba(24, 192, 237, 0.04);
+ }
+
+ table tr:nth-child(odd) {
+   background-color: rgba(240, 230, 230, 0.4);
+  }
+
+ td, th {
+   font-family: 'Reenie Beanie', cursive;
+   border-bottom: black;
+   width: 25%;
+   padding: 1rem;
+ }
+
+ th {
+   font-size: 26px;
+   margin-top: 0;
+ }
+
+ h1 {
+   font-family: 'Reenie Beanie', cursive;
+   text-align: center;
+   margin-top: 5%;
+ }
+
+ .content {
+   margin-left: 15%;
+   margin-right: 15%;
+ }
+ .add_task_head {
+   font-family: 'Reenie Beanie', cursive;
+   text-align: center;
+   margin-bottom: 1rem;
+   padding: 1rem;
+
+ }
+
+ .wave_one_header {
+   font-family: 'Reenie Beanie', cursive;
+   text-align: center;
+   margin-bottom: 1rem;
+   padding: 1rem;
+}
+
+.WHOOT {
+  font-family: 'Reenie Beanie', cursive;
+  font-size: 34px;
+  text-align: center;
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
+  padding: 1rem;
+  li {
+    margin-bottom: 3.5rem;
+  }
+}
+
+
+
+ h2 {
+   font-family: 'Reenie Beanie', cursive;
+   text-align: center;
+ }
+
+ form {
+   text-align: center;
+ }
+
+ button {
+   background-color: #fff;
+   border-radius: 1px;
+   border: none;
+   color: green;
+   padding: 2px 4px;
+   text-align: center;
+   text-decoration: none;
+   display: inline-block;
+   font-size: 16px;
+   margin: 1rem;
+   cursor: pointer;
+ }
+
+ textarea, input {
+   font-family: 'Reenie Beanie', cursive;
+   font-size: 24px;
+ }
+
+ .alert {
+   font-family: 'Reenie Beanie', cursive;
+   background-color: #E2DFD8;
+   font-size: 24px;
+   border-radius: 8px;
+   border: solid 2px;
+   color: black;
+   padding: 15px 20px;
+   text-align: center;
+   text-decoration: none;
+   display: inline-block;
+   margin-top: 1rem;
+   margin-bottom: 4rem;
+   cursor: pointer;
+   float: left;
+   ul {
+     padding-left: 0;
+   }
+ }
+
+ li {
+   list-style-type: none;
+ }
+
+ .button {
+   font-family: 'Reenie Beanie', cursive;
+   background-color: #2073e2;
+   border: none;
+   border-radius: 1rem;
+   color: white;
+   padding: 15px 32px;
+   text-align: center;
+   text-decoration: none;
+   display: inline-block;
+   font-size: 16px;
+   margin: 2px 1px;
+   margin-right: 20px;
+   cursor: pointer;
+     float: right;
+ }
+
+ .add_task {
+   text-align: center;
+   margin-top: 5%;
+ }
+
+ .edit_delete {
+   font-size: 20px;
+ }
+
+ ::-webkit-scrollbar {
+    -webkit-appearance: none;
+    width: 7px;
+ }
+ ::-webkit-scrollbar-thumb {
+    border-radius: 4px;
+    background-color: rgba(41, 102, 194, 0.2);
+    -webkit-box-shadow: 0 0 1px rgba(255,255,255,.5);
+ }
+
+ /* Track */
+ ::-webkit-scrollbar-track {
+     -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+     -webkit-border-radius: 10px;
+     border-radius: 10px;
+ }
```

Comment:
Nicely organized css. Watch out for excess whitespace, but other than that looks great!